libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts

```diff
@@ -176,7 +176,9 @@ export class SkyToggleSwitchComponent
   public labelComponents: QueryList<SkyToggleSwitchLabelComponent> | undefined;
 
   @HostBinding('style.display')
-  public display: string | undefined;
+  protected get display(): 'none' | undefined {
```

Comment:
Angular recommends against getters/methods in the template. Could this be refactored into a reusable pipe?

Reply:
Good call. Switched to a reusable host directive.